src/config/frontend.config.json

```diff
@@ -112,7 +112,7 @@
   "maxDirectDownloadSize": 5000000000,
   "metadataPreviewEnabled": true,
   "metadataStructure": "",
-  "multipleDownloadAction": "https://scicatfileserver.esss.dk/zip",
+  "multipleDownloadAction": "http:/127.0.0.1:3012/zip",
```

Comment:
We need to include ZipService image in the docker file later as well then

` /CI/E2E/docker-compose-local.yaml`